shell/components/Drawer/ResourceDetailDrawer/YamlTab.vue

```diff
@@ -24,9 +24,10 @@ const yamlComponent: any = useTemplateRef('yaml');
     :label="i18n.t('component.drawer.resourceDetailDrawer.yamlTab.title')"
     @active="() => yamlComponent?.refresh()"
   >
-    <CodeMirror
+    <ResourceYaml
```

Comment:
ResourceYaml implements the code folding for our resource pages so I switched over to using that component instead.